lib/cartopy/mpl/geoaxes.py

```diff
@@ -569,9 +569,8 @@ def _update_title_position(self, renderer):
     def __str__(self):
         return '< GeoAxes: %s >' % self.projection
 
-    def cla(self):
-        """Clear the current axes and adds boundary lines."""
-        result = super().cla()
+    def _clear(self):
```

Comment:
```suggestion
    def _cartopy_clear(self):
```

and update the calls below.

Reply:
we changed to `__clear` upstream to get the name mangling, but this is probably a good idea anyway!